vendor/github.com/containerd/containerd/plugin/plugin_go18.go

```diff
@@ -3,6 +3,6 @@
 package plugin
 
 func loadPlugins(path string) error {
-	// no plugin support to avoid binary size costs in balena-engine
+	// no plugin support to avoid binary size costs in balena
```

Comment:
Undid incorrect "manual" renaming of vendored repo (the contents of the `vendor` folder should be updated using the `vndr` tool)